config/default.yml

```diff
@@ -4,3 +4,6 @@ collectors:
     ansible-tower: 2
     azure: 2
     openshift: 2
+sync:
+  poll_time_seconds: 10 # seconds
```

Comment:
:+1: the suffix is nice to avoid confusion.  I think it makes the comment unnecessary.